### ISSUE TITLE
Make it work for the new site kiddo!

### ISSUE THREAD

**Issue by laughingclouds** [laughingclouds/Scrapia-World#4]
The site has recently been revamped. We knew it would happen :/

Make the code work for the new site.

**Comment by laughingclouds** [comment]
status update: I'm fairly close to making this run

For now here are a few things that work perfectly fine

1. Opening a new shell (as long as the current_chapter table has the required field)
2. setup function
    - installs the addons -> removes their welcome screens
    - logs in -> gets to the novel page

I haven't tested other functions/methods yet
but this is huge progress

**Comment by laughingclouds** [comment]
status update: I decided to take a different direction with how the program will behave initially.

Notable Additions:

- "novel profiling": No more searching for a particular chapter from the novel page when the program first runs. With the introduction of novel profiles (which is fancy name for "links of each and every chapter saved in a ``.json`` file") that part becomes redundant.
- More use of javascript: I've created a few javascript string creators (or something). Basically, you input tag names and some other information and a method will return a javascript doing operations with that tagname. This string will then be fed to the ``WebDriver`` object.

What more is left:

- A "loop": We need to create a loop that runs till the end of the final chapter has been reached.
- Remove links from **<>_toRead.json** for scraped chapters and append to **<>_read.json**.
- Some other stuff that I don't remember right now
